corehq/blobs/management/commands/import_blob_zip.py

```diff
@@ -17,14 +17,9 @@ def add_arguments(self, parser):
         parser.add_argument('zipname')
 
     def handle(self, zipname, **options):
+        # do we need to import BlobMeta records too?
```

Comment:
presumably yes - but that would be part of the SQL export / import unless you modify these to dump / import both